react/AsidedLayout/AsidedLayout.js

```diff
@@ -0,0 +1,30 @@
+import styles from './AsidedLayout.less';
+import React from 'react';
+import classnames from 'classnames';
+
+const defaultRenderAside = () => null;
+
+export default function AsidedLayout({ className, children, renderAside = defaultRenderAside, size, reverse }) {
+  return (
+    <div
+      className={classnames({
+        [className]: className,
+        [styles.root]: true,
+        [styles.reverse]: reverse
+      })}>
+      { reverse ?
```

Comment:
I'm not sure you would necessarily want asides content announced first. We can revisit once we have some example implementations